src/Nethermind/Nethermind.JsonRpc/Modules/Eth/EthRpcModule.cs

```diff
@@ -140,12 +142,108 @@ public ResultWrapper<Address> eth_coinbase()
             return ResultWrapper<UInt256?>.Success(0);
         }
 
-        [Todo("Gas pricer to be implemented")]
-        public ResultWrapper<UInt256?> eth_gasPrice()
+        private bool add_transactions_from_block_to_set(Block block, ref SortedSet<UInt256> sortedSet, UInt256 finalPrice, UInt256? ignoreUnder = null, long? maxCount = null)
```

Comment:
Method naming not within convention

Reply:
Are you referring to eth_gasPrice or add_transactions_from_block_to_set?